.github/workflows/dockerhub.yml

```diff
@@ -24,28 +24,28 @@ jobs:
         with:
           context: ./carma-messenger-ui/
           push: true
-          tags: usdotfhwastoldev/carma-messenger-ui:${{ github.ref_name }}-carma-messenger-ui         
+          tags: usdotfhwastoldev/carma-messenger-ui:develop
 
       - name: Build and Push carma-messenger-core Docker Image
         uses: docker/build-push-action@v3
         with:
           context: ./carma-messenger-core/
           push: true
-          tags: usdotfhwastoldev/carma-messenger-core:${{ github.ref_name }}-carma-messenger-core       
+          tags: usdotfhwastoldev/carma-messenger-core:develop

```

Comment:
This will push master changes as develop. I think for master we want to pull github tag for the release right.

Reply:
Correct. I wasn't sure how to retrieve the carma-system-XXXX version number, which is why I made it to push to develop. I can also make it not create image when master is merged, for which we can create a separate yml as well. What do you think?

Reply:
Ended up removing the master branch trigger because we need different "release*" related docker automation, which didn't seem like we had before anyways. Perhaps we can have Sai work on it

Reply:
@MishkaMN  I say remove all changes on this file since this CI process seems to work already.

Reply:
@paulbourelly999 
please take another look because the existing CI process for this one is actually incorrect. I stumbled upon it while changing the other CI yml. 
- Reason for develop tag change:
https://github.com/usdot-fhwa-stol/carma-messenger/blob/develop/carma-messenger-config/chevrolet_tahoe_2018/docker-compose-background.yml is the usage of the changed image for example. It just uses `develop` tag instead of `develop-carma-messenger-ui`. So I think it is correct to change it to just develop. 
- Reason for turning off the master branch trigger is because we don't push to `usdotfhwastoldev` org in dockerhub for master pushes. We push to `usdotfhwastol` org
Therefore the changes are non-breaking, so we should apply these changes now even if they may not be the complete fix to have developers take ownership of the tool and fix what's broken as we go.